### PR TITLE
Add QQ group link

### PR DESCRIPTION
**Pull request by hyzx86**

### CODE REVIEW

**Comment by Skrypt** [comment]
I've not been able to see this website. I get a blank page.
Though https://shang.qq.com/v3/index.html works for me.

**Comment by hyzx86** [comment]
> I've not been able to see this website. I get a blank page.
> Though https://shang.qq.com/v3/index.html works for me.

Maybe it's blocked by the firewall, but it works like this in China

image

**Comment by hyzx86** [comment]
The code is generate from here : https://qun.qq.com/join.html

**Comment by Skrypt** [comment]
Ok, I will leave others decide if we should add this to the documentation.
It's really specific for Chinese people though.

**Comment by hyzx86** [comment]
Yes, but now there is a strange problem, I don't know why the keywords' OrchardCore 'and' OrchardCore 'are blocked by the QQ official, so I have to change the name of the discussion group to' Orchard dotnet Core '. I have reported this problem to the Tencent official.

But that doesn't fit most people's search habits.
So I hope this link can be added to the official documentation of OC so that Chinese developers can find it directly

**Comment by Skrypt** [comment]
Yeah, it should be "Orchard Core". Maybe keep it that way and let us know when they removed the restriction.
I hope for you it's not a China moderation thing. Why would the "Orchard Core" term be an issue?

File: src/README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Orchard Core consists of two different targets:
 
 It’s important to note the differences between the framework and the CMS. Some developers who want to develop SaaS applications will only be interested in the modular framework. Others who want to build administrable websites will focus on the CMS and build modules to enhance their sites or the whole ecosystem.
 
+[![Orchard Core CN 中文讨论组](https://pub.idqqimg.com/wpa/images/group.png)](//shang.qq.com/wpa/qunwpa?idkey=48721591a71ee7586316604a7a4ee99d26fd977c6120370a06585085a5936f62)
```
Comment (HengzheLi) [inline src/README.md:12]: looks like it misses a "https:" in the front of "//shang.qq...."

Comment (hyzx86) [reply src/README.md:12]: This link will evoke the QQ, or jump to the QR code interface

Comment (hyzx86) [reply src/README.md:12]: The QQ group number is 877196442, welcome to join us 😎

Comment (HengzheLi) [reply src/README.md:12]: O. I see. It works.

**Comment by flew2bits** [comment]
Regarding OrchardCore being blocked, I have this same issue at work. It's because it contains the word "hardcore", and so it gets blocked.

**Comment by sebastienros** [comment]
I suggest to create a new group after the main content, or a section called "Local communities" and add the links.
In resources section

**Comment by hyzx86** [comment]
Hi @sebastienros ,  How about this ?

image

**Review by Skrypt — APPROVED** [review]

File: src/README.md [hunk 2]
```diff
@@ -14,6 +14,9 @@ It’s important to note the differences between the framework and the CMS. Some
 [![Documentation](https://readthedocs.org/projects/orchardcore/badge/)](https://docs.orchardcore.net/)
 [![Crowdin](https://badges.crowdin.net/orchard-core/localized.svg)](https://crowdin.com/project/orchard-core)
 
+#### Local communities
+
+[![Orchard Core CN 中文讨论组](https://pub.idqqimg.com/wpa/images/group.png)](//shang.qq.com/wpa/qunwpa?idkey=48721591a71ee7586316604a7a4ee99d26fd977c6120370a06585085a5936f62)
 ## Building Software as a Service (SaaS) solutions with the Orchard Core Framework
```
Comment (hishamco) [inline src/README.md:20]: Add new line before

**Review by hishamco — CHANGES_REQUESTED** [review]
Looks good after fix the format

**Review by hishamco — APPROVED** [review]

**Pull request merged**